File: 2271-rearrange-array-elements-by-sign/rearrange-array-elements-by-sign.cpp
class Solution {
public:
    vector<int> rearrangeArray(vector<int>& nums) {
        vector<int>p,n,v;
        for(int i = 0 ; i < nums.size();i++){
            if(nums[i]>0){
                p.push_back(nums[i]);
            }
            else{
                n.push_back(nums[i]);
            }
        }
        for(int i = 0 ; i < nums.size()/2;i++){
            v.push_back(p[i]);
            v.push_back(n[i]);
        }
        return v;
    }
};

File: 2271-rearrange-array-elements-by-sign/rearrange-array-elements-by-sign.py
class Solution:
    def rearrangeArray(self, nums: List[int]) -> List[int]:
        p=[]
        n=[]
        for i in range(0,len(nums)):
            if(nums[i]>0):
                p.append(nums[i])
            else:
                n.append(nums[i])
        f=[]
        for i in range(0,len(p)):
            f.append(p[i])
            f.append(n[i])
        return f